nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Analyze/CompatabilityChecker.cs

```diff
@@ -108,19 +108,18 @@ internal static bool PerformCheck(
 
         foreach (var d in dependencyGroups)
         {
-            var libItems = (await readers.ContentReader.GetLibItemsAsync(cancellationToken)).ToList();
-
-            foreach (var item in libItems)
-            {
-                tfms.Add(item.TargetFramework);
-            }
-
             if (!d.TargetFramework.IsAny)
             {
                 tfms.Add(d.TargetFramework);
             }
         }
 
+        var refItems = await readers.ContentReader.GetReferenceItemsAsync(cancellationToken);
```

Comment:
This was pulled out of the above loop because it's possible for a NuGet package to not list any dependency groups, but still have values in the `lib/` directory that need to be considered.